compatibility-verifier/compCheck.sh

```diff
@@ -173,29 +174,29 @@ fi
 
 # Setup initial cluster with olderCommit and do rolling upgrade
 startServices "$oldTargetDir"
-#$COMPAT_TESTER pre-controller-upgrade.yaml 1; if [ $? -ne 0 ]; then exit 1; fi
+#$COMPAT_TESTER $resourceDir pre-controller-upgrade.yaml 1; if [ $? -ne 0 ]; then exit 1; fi
```

Comment:
Provide absolute path to compat tester.
```suggestion
#$COMPAT_TESTER abspath($resourceDir/pre-controller-upgrade.yaml) 1; if [ $? -ne 0 ]; then exit 1; fi
```

Reply:
updated!